homeassistant/components/hydrawise/config_flow.py

```diff
@@ -22,25 +23,61 @@ class HydrawiseConfigFlow(ConfigFlow, domain=DOMAIN):
     """Handle a config flow for Hydrawise."""
 
     VERSION = 1
-
-    async def _create_entry(
-        self, api_key: str, *, on_failure: Callable[[str], ConfigFlowResult]
+    MINOR_VERSION = 2
```

Comment:
Either check `MINOR_VERSION` in `async_setup_entry`, or don't bump it and check if username + password are set. Now it's a weird mix between two migration styles.

Reply:
Fixed.